HTML/Ultimate Cipher condensed interactive (Fang).html

```diff
@@ -0,0 +1,1861 @@
+					<tr><th>2</th> <th>00010</th> <th>34</th><th>100010</th></tr>
+					<tr><th>3</th> <th>00011</th> <th>35</th><th>100011</th></tr>
+					<tr><th>4</th> <th>00100</th> <th>36</th><th>100100</th></tr>
+					<tr><th>5</th> <th>00101</th> <th>37</th><th>100101</th></tr>
+					<tr><th>6</th> <th>00110</th> <th>38</th><th>100110</th></tr>
+					<tr><th>7</th> <th>00111</th> <th>39</th><th>100111</th></tr>
+					<tr><th>8</th> <th>01000</th> <th>40</th><th>101000</th></tr>
+					<tr><th>9</th> <th>01001</th> <th>41</th><th>101001</th></tr>
+					<tr><th>10</th><th>01010</th><th>42</th><th>101010</th></tr>
+					<tr><th>11</th><th>01011</th><th>43</th><th>101011</th></tr>
+					<tr><th>12</th><th>01100</th><th>44</th><th>101100</th></tr>
+					<tr><th>13</th><th>01101</th><th>45</th><th>101101</th></tr>
+					<tr><th>14</th><th>01110</th><th>46</th><th>101110</th></tr>
+					<tr><th>15</th><th>01111</th><th>47</th><th>101111</th></tr>
+					<tr><th>16</th><th>10000</th><th>48</th><th>110000</th></tr>
+					<tr><th>17</th><th>10001</th><th>49</th><th>110001</th></tr>
+					<tr><th>18</th><th>10010</th><th>50</th><th>110010</th></tr>
+					<tr><th>19</th><th>10011</th><th>51</th><th>110011</th></tr>
+					<tr><th>20</th><th>10100</th><th>52</th><th>110100</th></tr>
+					<tr><th>21</th><th>10101</th><th>53</th><th>110101</th></tr>
+					<tr><th>22</th><th>10110</th><th>54</th><th>110110</th></tr>
+					<tr><th>23</th><th>10111</th><th>55</th><th>110111</th></tr>
+					<tr><th>24</th><th>11000</th><th>56</th><th>111000</th></tr>
+					<tr><th>25</th><th>11001</th><th>57</th><th>111001</th></tr>
+					<tr><th>26</th><th>11010</th><th>58</th><th>111010</th></tr>
+					<tr><th>27</th><th>11011</th><th>59</th><th>111011</th></tr>
+					<tr><th>28</th><th>11100</th><th>60</th><th>111100</th></tr>
+					<tr><th>29</th><th>11101</th><th>61</th><th>111101</th></tr>
+					<tr><th>30</th><th>11110</th><th>62</th><th>111110</th></tr>
+					<tr><th>31</th><th>11111</th><th>63</th><th>111111</th></tr>
+			</table>
+			
+			<h3>Edgework:</h3>
+				<font size="2">
+					<ol>
+					
+					<li>Step 1: # of ports is odd
+						<p style="text-align:center">
+							<code>
+							.........---------xxxxxxxx<br>
+							...---xxx...---xxx...---xx<br>
+							.-x.-x.-x.-x.-x.-x.-x.-x.-
+							</code>
+						</p>
+					</li>
+					<li>Step 2: Starting offset = Sum of SN Digits
+						<p style="text-align:center">
+							<code>00000000011111111112222222<br>12345678901234567890123456</code>
+						</p>
+					</li>
+				</ol>
+				</font>
+				<h3>Inverted Indigo Cipher Order:</h3>
+				<font size="2">
+				<ul>
+					<li>Step 1: Morbit Cipher (Keyword isn't effected)</li>
+					<li>Step 3: Hill Cipher</li>
+					<li>Step 2: Trifid Cipher</li>
+				</ul>
+				</font>
+			<table class = "alpha" style="width:55%;float:left">
+			<tbody>
+			<tr><th>In</th><th>AND</th><th>OR</th><th>XOR</th><th>NAND</th><th>NOR</th><th>XNOR</th><th>-></th><th><-</th></tr>
```

Comment:
please use &lt; and &gt; for less than and greater than symbols

Reply:
that looks more like a boolean operator than a larger/smaller than symbol